packages/flutter/lib/src/material/input_decorator.dart

```diff
@@ -1662,26 +1689,20 @@ class _InputDecoratorState extends State<InputDecorator> with TickerProviderStat
       ),
     );
 
-    final Widget prefix = decoration.prefix == null && decoration.prefixText == null ? null
-        : new AnimatedOpacity(
-        duration: _kTransitionDuration,
-        curve: _kTransitionCurve,
-        opacity: widget._labelIsFloating ? 1.0 : 0.0,
-        child: decoration.prefix ?? new Text(
-            decoration.prefixText,
-            style: decoration.prefixStyle ?? hintStyle
-        ),
+    final Widget prefix = decoration.prefix == null && decoration.prefixText == null ? null :
+      new _AffixText(
+        labelIsFloating: widget._labelIsFloating,
+        text: decoration.prefixText,
+        style: decoration.prefixStyle,
+        child: decoration.prefix,
       );
 
-    final Widget suffix = decoration.suffix == null && decoration.suffixText == null ? null
-        : new AnimatedOpacity(
-        duration: _kTransitionDuration,
-        curve: _kTransitionCurve,
-        opacity: widget._labelIsFloating ? 1.0 : 0.0,
-        child: decoration.suffix ?? new Text(
-            decoration.suffixText,
-            style: decoration.suffixStyle ?? hintStyle
-        ),
+    final Widget suffix = decoration.suffix == null && decoration.suffixText == null ? null :
+      new _AffixText(
+        labelIsFloating: widget._labelIsFloating,
+        text: decoration.suffixText,
+        style: decoration.suffixStyle,
```

Comment:
style should be `decoration.suffixStyle ?? hintStyle`, as before

Reply:
ups fixed it